pkg/resources/kafka/configmap.go

```diff
@@ -230,7 +233,7 @@ func generateListenerSpecificConfig(l *v1beta1.ListenersConfig, serverPasses map
 		listenerConfig = append(listenerConfig, fmt.Sprintf("%s://:%d", UpperedListenerName, eListener.ContainerPort))
 		// Add external listeners SSL configuration
 		if eListener.Type == v1beta1.SecurityProtocolSSL {
-			generateListenerSSLConfig(config, eListener.Name, "JKS", serverPasses, log)
+			generateListenerSSLConfig(config, eListener.Name, eListener.SSLClientAuth, "JKS", serverPasses, log)
```

Comment:
I think here the config in the argument (1beta1.ListenersConfig) contains the eListener.SSLClientAuth

Reply:
Oh yeah, you are right. I accidentally swaped this with the generateListenerSpecificConfig function.